File: Iniciante/1013 - O Maior/C#/1013 - O Maior.cs
using System;

class URI {
    static void Main() {
        string[] calculo = Console.ReadLine().Split(' ');
        // separa os valores para cada posição do array
        int A = Int32.Parse(calculo[0]);
        int B = Int32.Parse(calculo[1]); // cada variável recebe seu respectivo valor
        int C = Int32.Parse(calculo[2]);

        int maiorAB = (A + B + Math.Abs(A-B))/2;
        int maiorFinal = (maiorAB + C + Math.Abs(maiorAB-C))/2;

        Console.WriteLine(maiorFinal + " eh o maior");
    }
}

File: Iniciante/1013 - O Maior/Python/1013 - O Maior.py
# pega 3 valores na mesma linha e atribui às variáveis
A, B, C = input().split(' ')

# Converte os valores para os tipos necessários
A = int(A)
B = int(B)
C = int(C)

# cálculos aritméticos
maiorAB = (A+B+abs(A-B))/2
maiorAB = int((maiorAB+C+abs(maiorAB-C))/2)

# saída
print(maiorAB, 'eh o maior')